Wrong using statement added to Razor Pages work with a database tutorial's Visual Studio Code examples

Some of the VIsual Studio Code examples contain the following using statement:
```cs
using RazorPagesMovie.Data;
``` 
This line stops the example from building when going through the tutorial with Visual Studio Code.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 03221075-5dfa-d1b6-0552-3fe88abe1d03
* Version Independent ID: b8c7d824-35ca-68fb-e3be-839ae886657c
* Content: [Part 4, work with a database](https://learn.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/sql?view=aspnetcore-7.0&tabs=visual-studio&source=docs)
* Content Source: [aspnetcore/tutorials/razor-pages/sql.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/razor-pages/sql.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**


---
[Associated WorkItem - 36499](https://dev.azure.com/msft-skilling/Content/_workitems/edit/36499)

Here's an example of the error that occurs on build:
```sh
/home/colinkiama/RazorPagesMovie/Models/SeedData.cs(2,23): error CS0234: The type or namespace name 'Data' does not exist in the namespace 'RazorPagesMovie' (are you missing an assembly reference?) [/home/colinkiama/RazorPagesMovie/RazorPagesMovie.csproj]
```

--- a/aspnetcore/tutorials/razor-pages/model.md
+++ b/aspnetcore/tutorials/razor-pages/model.md
@@ -1,16 +1,14 @@
 ---
 title: Part 2, add a model
-author: rick-anderson
+author: wadepickett
 description: Part 2 of tutorial series on Razor Pages. In this section, model classes are added.
-ms.author: riande
+ms.author: wpickett
 ms.date: 11/04/2022
 monikerRange: '>= aspnetcore-3.1'
 ms.custom: contperf-fy21q2
 uid: tutorials/razor-pages/model
 ---
 # Part 2, add a model to a Razor Pages app in ASP.NET Core
-
-By [Rick Anderson](https://twitter.com/RickAndMSFT)
 
 :::moniker range=">= aspnetcore-7.0"
 
@@ -130,7 +128,7 @@ The `appsettings.json` file is updated with the connection string used to connec
 * Open a command shell to the project directory, which contains the `Program.cs` and `.csproj` files. On Windows, run the following command:
 
   ```dotnetcli
-  dotnet aspnet-codegenerator razorpage -m Movie -dc RazorPagesMovieContext -udl -outDir Pages\Movies --referenceScriptLibraries -sqlite
+  dotnet aspnet-codegenerator razorpage -m Movie -dc RazorPagesMovie.Data.RazorPagesMovieContext -udl -outDir Pages\Movies --referenceScriptLibraries -sqlite
   ```
 
   On non-Windows machines, run the following command:
@@ -145,7 +143,7 @@ The following table details the ASP.NET Core code generator options.
 | Option               | Description|
 | ----------------- | ------------ |
 | `-m`  | The name of the model. |
-| `-dc`  | The `DbContext` class to use. |
+| `-dc`  | The `DbContext` class to use including namespace. |
 | `-udl` | Use the default layout. |
 | `-outDir` | The relative output folder path to create the views. |
 | `--referenceScriptLibraries` | Adds `_ValidationScriptsPartial` to Edit and Create pages |

--- a/aspnetcore/tutorials/razor-pages/model.md
+++ b/aspnetcore/tutorials/razor-pages/model.md
@@ -134,7 +134,7 @@ The `appsettings.json` file is updated with the connection string used to connec
   On non-Windows machines, run the following command:
 
   ```dotnetcli
-  dotnet aspnet-codegenerator razorpage -m Movie -dc RazorPagesMovieContext -udl -outDir Pages/Movies --referenceScriptLibraries -sqlite
+  dotnet aspnet-codegenerator razorpage -m Movie -dc RazorPagesMovie.Data.RazorPagesMovieContext -udl -outDir Pages/Movies --referenceScriptLibraries -sqlite
   ```
 
 <a name="codegenerator"></a>

